src/GafferScene/ShaderPlug.cpp

```diff
@@ -48,31 +48,23 @@ using namespace Gaffer;
 using namespace GafferScene;
 
 //////////////////////////////////////////////////////////////////////////
-// Internal utilities
+// ShaderPlug
 //////////////////////////////////////////////////////////////////////////
 
-namespace
-{
+IE_CORE_DEFINERUNTIMETYPED( ShaderPlug );
 
-const GafferScene::Shader *shader( const ShaderPlug *plug )
+
+const Gaffer::Plug *ShaderPlug::getShaderOutPlug() const
```

Comment:
I was wondering why some getters were just noun phrases but others had getXYZ. Didn't spot the rule thanks for the explanation.